packages/node-provider/src/types.ts

```diff
@@ -1,12 +1,25 @@
+export type NodeMessageReceivedCallback = (message: NodeMessage) => void;
+
 export interface INodeProvider {
-  onMessage(callback: (message: NodeMessage) => void);
+  status: NodeProviderStatus;
+  onMessage(callback: NodeMessageReceivedCallback);
   postMessage(message: NodeMessage);
```

Comment:
When me and @snario whiteboarded CFjs we concluded that this method creates confusion with the browser PostMessage protocol used between NodeProvider and the Playground. 

"sendMessage" or something similar would be better.

Reply:
`sendMessage` is cool with me